lib/plugins/features/movies/controller.js

```diff
@@ -10,6 +11,7 @@ exports.create = async (payload) => {
 };
 
 exports.find = async (query) => {
+  const Movies = Movie.collection();
```

Comment:
Not saying this pattern is wrong, but in `lob-api` and other places we don't explicitly use collections like this. We just initialize a new model like you did in `exports.create`. 

So this would be like 
```
return new Movies().query((qb) => {
...
})
.fetch(...)
```

Again, your pattern works but to be consistent with the way its done in our codebases I'd do it the other way to avoid confusion